Tales/Date.php

```diff
@@ -0,0 +1,187 @@
+<?php
+/**
+ * PHPTal Tale modifiers: date utilities.
+ *
+ * @copyright 2022 Namesco Limited
+ */
+
+namespace Ztal\Tales;
+
+/**
+ * PHPTal Tale modifiers: date utilities.
+ *
+ * Useful date utility routines within a template.
+ */
+final class Date implements \PHPTAL_Tales
+{
+	/**
+	 * Current timestamp.
+	 *
+	 * @var integer
+	 */
+	protected static int $_currentTs;
+
+	/**
+	 * Tal extension to handle a "between" date comparison (fromDate <-> now <-> toDate).
+	 *
+	 * Note that the exact time for a given date is midnight (the very start of the day), so the date 2022-02-25 is
+	 * actually 2022-02-25 00:00:00. Date ranges should take this into account
```

Comment:
I’d personally implement this so that the second date has +1 day added to it so that “between” effectively works the same as BETWEEN in SQL.

Reply:
Yep, that makes sense. Had that the same thought at the back of my mind.